apps/mdn/mdn-aws/k8s/Makefile

```diff
@@ -203,6 +206,9 @@ export NEW_RELIC_MONITOR_MODE ?= 1
 export NEW_RELIC_LOG ?= stderr
 export NEW_RELIC_BROWSER_MONITOR_ENABLE ?= false
 export NEW_RELIC_LABELS ?= Project:MDN;Cluster:${K8S_CLUSTER_SHORT_NAME};Namespace:${K8S_NAMESPACE}
+export NEW_RELIC_HIGH_SECURITY ?= true
```

Comment:
IIUC, this is a kind of anti-tracking protection. I'd be inclined to make it non-configurable and just hard-code the true in the template file.

Reply:
Good point, will do!